kubectl-minio/cmd/tenant-upgrade.go

```diff
@@ -138,10 +148,36 @@ func (u *upgradeCmd) run(args []string) error {
 	return nil
 }
 
-func upgradeTenant(client *operatorv1.Clientset, t *miniov1.Tenant) error {
-	if _, err := client.MinioV1().Tenants(t.Namespace).Update(context.Background(), t, v1.UpdateOptions{}); err != nil {
-		return err
+func (u *upgradeCmd) ask() bool {
+	reader := bufio.NewReader(os.Stdin)
+	for {
+		s, _ := reader.ReadString('\n')
+		s = strings.TrimSuffix(s, "\n")
+		s = strings.ToLower(s)
+		if len(s) > 1 {
+			fmt.Fprintln(os.Stderr, "Please enter Y or N")
+			continue
+		}
+		if strings.Compare(s, "n") == 0 {
+			return false
+		} else if strings.Compare(s, "y") == 0 {
+			break
+		} else {
+			continue
+		}
+	}
+	return true
+}
+
+func (u *upgradeCmd) upgradeTenant(client *operatorv1.Clientset, t *miniov1.Tenant, c, p string) error {
+	fmt.Printf(color.Bold(fmt.Sprintf("\nAre you sure to update Tenant '%s/%s' from %s to %s? [Y/N]:", t.ObjectMeta.Name, t.ObjectMeta.Namespace, c, p)))
```

Comment:
"Are you sure you want to upgrade Tenant..."

Also wondering if we should state something like "You cannot downgrade to an earlier version of MinIO after upgrading" (assuming we have that restriction).